huaweicloud/services/nat/resource_huaweicloud_nat_private_transit_ip.go

```diff
@@ -83,90 +83,194 @@ func ResourcePrivateTransitIp() *schema.Resource {
 	}
 }
 
+func buildCreatePrivateTransitIpBodyParams(d *schema.ResourceData, epsId string) map[string]interface{} {
+	transitIpBodyParams := map[string]interface{}{
+		"virsubnet_id":          d.Get("subnet_id"),
+		"ip_address":            utils.ValueIgnoreEmpty(d.Get("ip_address")),
+		"enterprise_project_id": utils.ValueIgnoreEmpty(epsId),
+		"tags":                  utils.ValueIgnoreEmpty(utils.ExpandResourceTags(d.Get("tags").(map[string]interface{}))),
```

Comment:
Pay attention to maintaining consistency with the source code logic. Think about what the default value is.